File: SQLite_Example/app/src/main/java/com/example/sqlite/juan/helpers/DataBaseHelper.java
package com.example.sqlite.juan.helpers;

import android.content.Context;
import android.database.sqlite.SQLiteDatabase;
import android.database.sqlite.SQLiteOpenHelper;

public class DataBaseHelper extends SQLiteOpenHelper {

    public DataBaseHelper(Context context) {
        super(context, Constants.DB, null, 1);
    }

    @Override
    public void onCreate(SQLiteDatabase sqLiteDatabase) {
        sqLiteDatabase.execSQL(Constants.create_table_usuarios);
    }

    @Override
    public void onUpgrade(SQLiteDatabase sqLiteDatabase, int i, int i1) {
        sqLiteDatabase.execSQL(Constants.delete_table_usuarios);
        onCreate(sqLiteDatabase);
    }

}

File: SQLite_Example/app/src/main/java/com/example/sqlite/juan/helpers/DataBaseHelper.kt
package com.example.sqlite.juan.helpers

import android.content.Context
import android.database.sqlite.SQLiteDatabase
import android.database.sqlite.SQLiteOpenHelper

class DataBaseHelper(context: Context?) :
    SQLiteOpenHelper(context, Constants.DB, null, 1) {

    override fun onCreate(sqLiteDatabase: SQLiteDatabase) {
        sqLiteDatabase.execSQL(Constants.create_table_usuarios)
    }

    override fun onUpgrade(sqLiteDatabase: SQLiteDatabase, i: Int, i1: Int) {
        sqLiteDatabase.execSQL(Constants.delete_table_usuarios)
        onCreate(sqLiteDatabase)
    }
}